tx-indexer/src/config.rs

```diff
@@ -369,4 +434,116 @@ impl ScyllaDBManager {
         .await?;
         Ok(())
     }
+
+    pub(crate) async fn add_transaction_process(
+        &self,
+        transaction_details: readnode_primitives::CollectingTransactionDetails,
+    ) -> anyhow::Result<()> {
+        let transaction_hash = transaction_details.transaction.hash.clone().to_string();
+        let transaction_details = transaction_details.try_to_vec()?;
+        Self::execute_prepared_query(
+            &self.scylla_session,
+            &self.add_transaction_process,
+            (transaction_hash, transaction_details),
+        )
+        .await?;
+        Ok(())
+    }
+    pub(crate) async fn add_receipt_process(
+        &self,
+        transaction_hash: &str,
+        indexer_execution_outcome_with_receipt: near_indexer_primitives::IndexerExecutionOutcomeWithReceipt,
+    ) -> anyhow::Result<()> {
+        Self::execute_prepared_query(
+            &self.scylla_session,
+            &self.add_receipt_process,
+            (
+                transaction_hash,
+                indexer_execution_outcome_with_receipt
+                    .receipt
+                    .receipt_id
+                    .to_string(),
+                indexer_execution_outcome_with_receipt
+                    .receipt
+                    .try_to_vec()?,
+                indexer_execution_outcome_with_receipt
+                    .execution_outcome
+                    .try_to_vec()?,
+            ),
+        )
+        .await?;
+        Ok(())
+    }
+
+    pub(crate) async fn get_transactions_in_process(
```

Comment:
Consider replacing `process` with `progress`. I believe it reflects what we are doing better. @telezhnaya wdyt?